wiki/Getting-Started.md

```diff
@@ -0,0 +1,118 @@
+So you're interested in trying this out. Start by cloning the repo and checking out the 'WorkflowExample' scheme. This should give you a decent idea of how the library works conceptually.
```

Comment:
This only conceptually tells you how it works? Conceptually as a word really bothers me here. I keep rereading it and it's a little off-putting about everything that comes next.